Core/src/OffConverter/OffModel.cpp

```diff
@@ -26,7 +26,7 @@ void OpenInfraPlatform::Core::OffConverter::OffModel::reset()
 
 void OpenInfraPlatform::Core::OffConverter::OffModel::addVertex(buw::Vector3d vertex)
 {
-	geometry_.vertices.push_back;
+	geometry_.vertices.push_back(vertex);
 	geometry_.UpdateBBox();
```

Comment:
I suggest to update the BBox only after all vertices have been read in - otherwise you have a O(n) repretition every time you add a vertex. Perhaps another idea would be to push_back in the reading function and not even allow for addVertex?

Reply:
All vertices now get added at the same time and UpdateBBox only gets called once at the end 
Resolved: bb01365